static/app/views/replays/detail/focusArea.tsx

```diff
@@ -18,13 +19,22 @@ type Props = {
   memorySpans: MemorySpanType[] | undefined;
 };
 
+const DEFAULT_TAB = 'performance';
+const TABS = new Set(['performance', 'errors', 'tags', 'memory']);
```

Comment:
turning `TabBarId` into a ts enum would make it so that we don't need to type the `new Set` here.

tl/dr:
```
// transforms into an object: `{CONSOLE: 'console', ... }`
enum TABS {
  CONSOLE = 'console',
  PERFORMANCE = 'performance',
  ERRORS = 'errors',
  TAGS = 'tags',
  MEMORY = 'memory',
};

function isTabBarId(tab: string): tab is TABS {
  return tab.toUpperCase() in TABS;
}
```

[ts playground](https://www.typescriptlang.org/play?#code/PTAEBcE8AcFNQCoEMBGAhJAnAkgE1ALygDkAxgPYB2AzuQDazGgA+JcmAZuZgLZKWlGLErEyZu1Jq2LgkAc0nDiPWD26RiAbgBQIUBRrhEAQTQBlQqEqwA7qDOxwACgDaZKrQbEANG1FdefkEfETEJEJl5SV9lVXViAF0ASh1tWEoAVx4Tc1AAb21QUABhAHkAOTNSgBkAUUt3GnpGb0LQAAVagCUAMVKugFljcuL6omJ2AL4BFrburv6LcdFxTGi2hGMAcSWSWQUfNoHagf6ATQaVNUwNVoBfVI4MgXAASypQV+pkdCw8J1kKAAXKBqOBMK9KHIkiDAZ9qDkLAUiphHBlMJQIKgAHTgcgAVWg7GKSGosCcSU+mM25h0d202gMYNAAAtSSzLDZIbhyDZsXRyKQkG8qNi2dQWdjUdA6EhBE5iABiCLEFKMjxGQE9cQ8AAS7MsXx+GBwuCc4pZlIA-KyDSCaWZsZ1ev0hiNao9nqQRZiuOQAah7aYzJTkfoPM1+eQ5Aq4V8QoC1fTtH6AyhteQ9ey1amHdiypUarUUkA

)